paddlenlp/embeddings/token_embedding.py

```diff
@@ -218,19 +224,24 @@ def _extend_vocab(self, extended_vocab_path, vector_np, pad_vector,
     def set_trainable(self, trainable):
         """
         Set the weight of embedding can be trained.
```

Comment:
Set the weight of embedding can be trained.
->
Whether or not to set the weights of token embedding to be trainable.